client/lib/post-normalizer/rule-wait-for-images-to-load.js

```diff
@@ -21,7 +21,7 @@ import debugFactory from 'debug';
 const debug = debugFactory( 'calypso:post-normalizer:wait-for-images-to-load' );
 
 function convertImageToObject( image ) {
-	return pick( image, [ 'src', 'naturalWidth', 'naturalHeight' ] );
+	return pick( image, [ 'src', 'naturalWidth', 'naturalHeight', 'alt' ] );
```

Comment:
this is the `Image` object from `imageForUrl`, not the `img` tag in the dom. You'll need to alter `imageForUrl` to pass along the `alt` (and probably the `title` too?) and set those on the image object to pull 'em back off.